controllers/cloud.redhat.com/providers/featureflags/localfeatureflags.go

```diff
@@ -41,18 +44,16 @@ var LocalFFDBPVC = rc.NewSingleResourceIdent(ProvName, "ff_db_pvc", &core.Persis
 // LocalFFDBSecret is the ident referring to the local Feature Flags DB secret object.
 var LocalFFDBSecret = rc.NewSingleResourceIdent(ProvName, "ff_db_secret", &core.Secret{})
 
-// LocalFFSecret is the ident referring to the local Feature Flags secret object.
-var LocalFFSecret = rc.NewSingleResourceIdent(ProvName, "ff_secret", &core.Secret{})
-
 type localFeatureFlagsProvider struct {
 	providers.Provider
 }
 
 // NewLocalFeatureFlagsProvider returns a new local featureflags provider object.
 func NewLocalFeatureFlagsProvider(p *providers.Provider) (providers.ClowderProvider, error) {
 	p.Cache.AddPossibleGVKFromIdent(
-		LocalFFDBDeployment,
+		LocalFFDeployment,
 		LocalFFService,
+		LocalFFSecret,
 		LocalFFDBDeployment,
```

Comment:
Is this supposed to be here ? why is it necessary to update the featureflags service ?

Reply:
It was missing - as I went through fixing other stuff I noticed that it was missing and added it

Reply:
OK, thanks for clarifying and for fixing the bug!